CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Given a version number `MAJOR.MINOR.PATCH`, we increment the:
 - Add `list --run-order` flag to list stacks in the order they would be executed.
 - Add support for deployment syncing to script commands.
 - Add `disable_safeguards` configuration option and CLI flag.
+- Add `terramate.config.generate.hcl_magic_header_comment_style` option for setting the generated comment style.
```

Comment:
FYI the name of this field was not set in stone, in the refinement meeting we discussed that maybe we could also come up with a shorter name, i.e. `hcl_header_prefix`.